search/processor.cpp

```diff
@@ -294,12 +253,12 @@ void Processor::SetRankPivot(m2::PointD const & pivot)
 
 void Processor::SetLanguage(int id, int8_t lang)
 {
-  m_keywordsScorer.SetLanguage(GetLangIndex(id), lang);
+  m_ranker.m_keywordsScorer.SetLanguage(GetLangIndex(id), lang);
```

Comment:
May be it worth to protect keywords scorer, and create a bunch of public methods in Ranker instead?